extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,400 @@
+As the mind will perceive object brightess compared to the background, ie 100 vs 10, it is vital to have a known higher dynamic range in order to achieve deterministic display output.     
+
+Apart from being widely supported and used in the TV / movie industry the perceptual quantizer is also embraced by the gaming community, with support in the engines from some of the major game companies.  
+For instance, game engines Frostbite and Lumberyard and also in specific games such as Destiny 2 and Call Of Duty.  
+
+
+
+## Internal range of light contribution values
+
+This section describes how light contribution values shall be handled.  
+
+When the KHR_displaymapping_pq extension is used all lighting and pixel calculations shall be done using the value 10000 (cd / m2) as the maximum ouput brightness.  
+This does not have an impact on color texture sources since they define values as contribution factor.  
+The value 10000 cd / m2 for an output pixel with full brightness is chosen to be compatible with the Perceptual Quantizer (PQ) used in the SMPTE ST 2084 transfer function.  
+
+When using this extension light contribution values shall be aligned to account for 10000 cd/m2 as max luminance, meaning that the dynamic range is 0 to 10000.    
+This means that content creators shall be aware of 10000 cd/m2 as the maximum brightness value range.  
+It does not mean that the display will be capable of outputing at this light luminance.  
+
+
+
+## Exporter considerations for light contribution values
+
+This section describes how an exporter shal handle lightsources, such as point, directional or environment lights, that are saved with the model.  
+
+A content creation tool supporting this extension shall sum upp light contribution for a scene before exporting to glTF, this can be a naive addition of all lights included in the scene that adds max values together.  
+If scene max light contribution is above 10000 cd / m2 there is a choice to either downscale light values before export or to set `sceneAperture` to `AUTO`  
+This means that implementations will calculate max light intensity for the scene and use as a scale factor to keep all light contribution within 0 - 10000 cd / m2.  
+
+Light contribution values above 10000 cd / m2 is strongly discouraged and will be clamped by implementations if not adjusted by scene aperture.    
+
+
+## Displaymapping
+
+This section describes how the mapping of internal, scene linear light, values to display output values in the range [0 - 1.0] shall be done.  
+
+Displaymapping will be done different depending on wether the output is an HDR or SDR capable display.  
+If the output type is unknown then it shall be considered to be SDR, and the SDR ootf shall be used.  
+
+### To HDR capable display
+
+To convert from internal values (linear scene light) to the non-linear output value in the range 0.0 - 1.0 the PQ EOTF shall be used.  
```

Comment:
The EOTF will happen in the display (or display pipeline). The role of the renderer for HDR display is to output PQ encoded data using the OETF.

Reply:
Good catch - that is simply a typo